srcpkgs/kleopatra/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'kleopatra'
+pkgname=kleopatra
+version=20.12.3
+revision=1
+archs="x86_64"
```

Comment:
There shouldn't be any inherent reason why `kleopatra` shouldn't work on any other arch.
So remove and either fix problem that might crop up or ask for help.

Reply:
It seems to work on x86_64, x86_64-musl and i686 only. I will add this to archs and hope it is fine.

Reply:
If it isn't working on ARM, it's likely due to something missing for a cross build, and not anything special about x86.

Reply:
Remove the line entirely so we can see the errors that it spits out.

Reply:
Sorry, i deleted my comment now because I did see it by myself. There is something wrong with qmake. Maybe I need to add as a dependency.

Reply:
I did remove the line.

Reply:
```
  "/usr/lib/qt5/bin/qmake"

  but this file does not exist.  Possible reasons include:
```
Yep, that message says that qmake is needed, which would be provided by `qt5-qmake`.

Reply:
I did add qt5-qmake as a dependency ... But still won't work. Is there something messed up with qt5-qmake in armv7l ?

Reply:
`/usr/armv7l-linux-gnueabihf/usr/bin/msgfmt: /usr/armv7l-linux-gnueabihf/usr/bin/msgfmt: cannot execute binary file`
whatever